austrakka/components/project/setup/__init__.py

```diff
@@ -0,0 +1,92 @@
+from typing import List
+import click
+from austrakka.components.project.setup.funcs import \
+    add_field_project, \
+    set_merge_algorithm_project, \
+    add_provision_project, get_dataset_provision_list, get_project_field_list, \
+    remove_project_field, remove_project_provision, update_project_provision
+from austrakka.utils.cmd_filter import hide_admin_cmds
+from austrakka.utils.options import \
+    opt_field_name, \
+    opt_merge_algorithm, \
+    opt_field_and_source, \
+    opt_abbrev, opt_prov_id
+from austrakka.utils.output import table_format_option
+
+
+# pylint: disable=duplicate-code
+@click.group()
+@click.pass_context
+def setup(ctx):
+    """Commands to upload project datasets"""
```

Comment:
This is an error - the command group `setup` has the same text as `dataset`.

However I actually think (as per slack discussion) we should split these commands into two command groups: `provision` and `field`, and do away with `setup` altogether